### DISCUSSION TITLE
服务使用一段时间后报milvus服务连不上

### DISCUSSION THREAD

**Discussion by WinterWatcher** [milvus-io/milvus#35557] [Q&A and General discussion]
![image](https://github.com/user-attachments/assets/c11af8a2-1f4c-428b-b8f8-9261da13ef0f)

**Comment by WinterWatcher** [comment]
求助！

**Comment by yhmo** [comment]
端口是不是映射错了，看你连的端口是8530

**Reply by WinterWatcher** [reply]
改了容器端口映射

**Reply by yhmo** [reply]
检查一下容器是不是挂了。用 docker logs [container id] > 1.log 命令把日志拿出来

**Reply by WinterWatcher** [reply]
容器没有问题，用attu能连，就是服务连不上

**Reply by yhmo** [reply]
No servers found for xxx 这个报错很少见，感觉跟网络环境有关，你在172.16.2.57这台机器内部试试用sdk连接能否连上

**Reply by WinterWatcher** [reply]
我这边先尝试一下

**Reply by WinterWatcher** [reply]
感谢您的帮助

**Comment by xiaofan-luan** [comment]
是否开了TLS？可以telnet端口看看是否联通？

**Reply by WinterWatcher** [reply]
我这边先尝试一下

**Reply by WinterWatcher** [reply]
感谢您的帮助
